data/kahet/kahet missions.txt

```diff
@@ -721,3 +721,1128 @@ event "pacified ka'sei"
+			`	"Oh, definitely! While the data we could gather was not as much as I hoped, we now have a rather clear connection between Patir and Chanai, as well as between the station and those peculiar beings. That, alone, is invaluable for our mission.`
+			label next
+			`	"During the trip back, we evaluated the options at our disposal. As you may guess, the most obvious one would be to explore further into the station; however, it was not as attractive as previously thought. While it may have been used to influence Patir, it is clear that it now sits silent and powerless, as if missing something that can bring it back to life. We could find more about its design perhaps, but the chances of using it on our own within a reasonable time scale are low.`
+			`	"We have enough consumables to last for many weeks still; we started rationing as soon as we arrived in this sector, and we are prepared to reduce them further if necessary. Nevertheless, we need to take a more direct approach if we want results. Approaching the species that inhabits this cluster is essential at this point, and doing so immediately would be advisable."`
+			choice
+				`	"What do you suggest?"`
+					goto plan
+				`	"Are you proposing a first contact of some sort?"`
+					goto contact
+				`	"Are they behind the Chanai station?"`
+			`	"They could be. The internal design of the station is an odd fit for the asteroids' size and characteristics, but we do know that they have previously visited it and modified it in some way.`
+				goto plan
+
+			label contact
+			`	"In a way. We have been considering a close approach to one of those beings ever since we first saw them, and your discovery of the planetoid in Lathia opens up the opportunity for a surface expedition - gathering samples, analyzing its reaction, and perhaps yes, if they are intelligent and the conditions are right, a true first contact between our two species could happen.`
+				goto lab
+			label plan
+			`	"We have been considering a close approach to one of those beings ever since we first saw them, and your discovery of the planetoid in Lathia opens up the possibility for a surface expedition. Due to its size, landing with the <ship> should be possible, and we could even explore the planetoid in our suits if the surface is safe. Perhaps, if these beings are truly intelligent, this could be the first contact between our two species.`
+			label lab
+			`	"Meet us in the lab in a few hours, and we will have everything ready for the trip. With a bit of luck, most of the instruments we used on Chanai will be a good fit for this mission as well."`
+		
+mission "Ka'het: Patir Mystery 12B"
+	name "Satellite deployment"
+	description "Deploy two probes in <stopovers> to keep an eye on incoming space rocks."
+	source "Builder Settlement"
+	passengers 1
+	cargo "Remnant Probes" 3
+	waypoint "Mitera"
+	waypoint "Thepa"
+	to offer
+		has "Ka'het: Patir Mystery 11: done"
+		not "Ka'het: Patir Mystery 12A: accepted"
```

Comment:
This is not a condition that will be automatically set and it doesn't look like you are setting it manually.
I'm guessing you want this mission to offer immediately after declining mission 12A, and not to offer at all if mission 12A is accepted?
In order to achieve that, I suggest something like:
```
mission "whatever 12B"
	to accept
		has "whatever 12A: offered"
		has "whatever 12A: declined"
```